en/pyvespa/connect-to-vespa-instance.md

```diff
@@ -0,0 +1,79 @@
+---
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Connect to a running Vespa instance"
```

Comment:
I think a better title is "Query Vespa from python" - or something like that, "connecting" is not that sexy and is a small part of it...

Reply:
I agree with the sexy part, but the problem is that this tutorial shows how to connect to a running Vespa instance. The query part is there to show that it works. There will be other tutorials where you deploy your own app and query it, so the differentiation between them is how you connect to the app and not the query part.

I will merge this, and we can reevaluate the title when we include the other tutorials.